### PR TITLE
Clarify curvature drive comments, fix Java code example

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: source/docs/software/actuators/wpi-drive-classes.rst [hunk 1]
```diff
@@ -278,30 +278,37 @@ Joystick construction.
...
-that is used to engage a sub-mode that allows for turning in
-place. This example shows how to use a single joystick with
-the Curvature mode. Construction of the objects has been
-omitted, for above for drivetrain construction and here for
-Joystick construction.
+rate. The difference is that the rotation control input
+controls the radius of curvature instead of rate of
+heading change, much like the steering wheel of a car. This
+mode also supports turning in place, if necessary, through
+an additional parametr.
```
Comment (AustinShalit) [inline source/docs/software/actuators/wpi-drive-classes.rst:285]:
```suggestion
an additional parameter. 
```

File: source/docs/software/actuators/wpi-drive-classes.rst [hunk 2]
```diff
@@ -180,29 +180,39 @@ Drive Modes
...
 
-Like Arcade Drive, the Curvature Drive mode is used to control the drivetrain using speed/throttle and rotation rate. The difference is that the rotation control is attempting to control radius of curvature instead of rate of heading change. This mode also has a quick-turn parameter that is used to engage a sub-mode that allows for turning in place. This example shows how to use a single joystick with the Curvature mode. Construction of the objects has been omitted, for above for drivetrain construction and here for Joystick construction.
+Like Arcade Drive, the Curvature Drive mode is used to control the drivetrain using speed/throttle and rotation rate. The difference is that the rotation control input controls the radius of curvature instead of rate of heading change, much like the steering wheel of a car. This mode also supports turning in place through an additional parameter.
```
Comment (Daltz333) [inline source/docs/software/actuators/wpi-drive-classes.rst:189]: What is this additional parameter?

**Review by Daltz333 — CHANGES_REQUESTED** [review]

File: source/docs/software/actuators/wpi-drive-classes.rst (hunk 2 above)
Comment (mcm001) [reply source/docs/software/actuators/wpi-drive-classes.rst]: this better?

Comment (Daltz333) [reply source/docs/software/actuators/wpi-drive-classes.rst]: I was more wanting a code example.

Comment (mcm001) [reply source/docs/software/actuators/wpi-drive-classes.rst]: There is one?

Comment (Daltz333) [reply source/docs/software/actuators/wpi-drive-classes.rst]: Oh lol

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**